parcellab/microservice/values.yaml

```diff
@@ -46,6 +46,87 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+##
+## Envoy Gateway Resources
+##
+
+# HTTPRoute - Configure routing through Envoy Gateway
+httproute:
+  enabled: false
+  # parentGateway: gateway-api
+  # parentGatewayNamespace: envoy-gateway
+  # hosts:
+  #   - myapp.gateway.test.parcellab.dev
+  # path: /
+
+# ReferenceGrant - Allow cross-namespace references
+referenceGrant:
```

Comment:
Are there settings for the referenceGrant, that need to be specified differently ? Meaning, isn't it possible to derive the httpRoutes and referenceGrant entirely from the service settings ?